src/main/resources/scripts/cleanup/non_prod_primary_and_replica_data_cleanup.sql

```diff
@@ -0,0 +1,101 @@
+--
+-- RWA-2482 cleanup old tasks from primary task database
+--
+--Example for calling the function
+--select cft_task_db.task_cleanup_between_dates_primary('2023-04-04 11:00'::TIMESTAMP,'2023-04-04 11:01'::TIMESTAMP);
+--
+
+CREATE OR REPLACE FUNCTION cft_task_db.task_cleanup_between_dates_primary(created_time_from TIMESTAMP,created_time_to TIMESTAMP)
+RETURNS VOID
+LANGUAGE plpgsql
+AS $function$
+DECLARE
+    task_id_var TEXT;
+BEGIN
+    -- Loop through the rows of the tasks table
+    FOR task_id_var IN SELECT task_id FROM cft_task_db.tasks WHERE created >= create_from AND created <= create_to order by created limit 2000 LOOP
```

Comment:
Don't think this will work.  You have to update the names of the params you're using.

Reply:
Oh no... I missed this... I made the changes now Ravi.... Please verify...